<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="ajouteretmodifier">
     <attribute name="title">
      <string>ajouter et modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero d'evenement</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date / Time</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Emplacement</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Responsable</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emplacement">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="responsable">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="numero">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">DateTimeEdit{
color :rgb(255, 170, 255);
background-color : #b0e0e6;
font-size : 18px;
border-style : outset;
border-radius:10px;
background : #b0e0e6;
border-style : outset;
border-radius:10px;
}
QPushButton:pressed{
background-color :rgb(241,210,203);
border-style:inset;
}</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="date">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="fian">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>fian√ßailles</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="anniv">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>anniversaire</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="souten">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>110</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>soutenance</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="babysho">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>110</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>baby shower</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="mariage">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>mariage</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color :#000080;
background-color : #b0e0e6;
font-size : 18px;
border-style : outset;
border-radius:10px;
background : #b0e0e6;
border-style : outset;
border-radius:10px;
}
QPushButton:pressed{
background-color :rgb(241,210,203);
border-style:inset;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color :#000080;
background-color : #b0e0e6;
font-size : 18px;
border-style : outset;
border-radius:10px;
background : #b0e0e6;
border-style : outset;
border-radius:10px;
}
QPushButton:pressed{
background-color :rgb(241,210,203);
border-style:inset;
}</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero d'evenement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>generer excel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>260</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>generer un pdf </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableevent">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>301</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox1">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>1</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>tyype</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>datetime</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Numero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Emplacement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>responsable</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>gerer code QR</string>
     </attribute>
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>300</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>la localisation:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>stat</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
